client/post-editor/editor-sidebar/feedback-list.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import Gravatar from 'components/gravatar';
+
+class FeedbackList extends PureComponent {
```

Comment:
Since I'll be wrapping with the Redux helper as a next step, I'll make a note to switch to `Component` in those cases. Does that work for you?